projects/mysadna2/src/app/projects/projects.component.html

```diff
@@ -1 +1,10 @@
-<h2>Projects</h2>
+<div class="projects">
+  <h2>Projects</h2>
+  <ul style="list-style:none">
+    <li *ngFor="let project of projects">
+      <a routerLink="/project-details/{{project.projectId}}">
+        {{project.projectName}}
+      </a>
```

Comment:
> I keep trying to center the text in projects, but it turns out lopsided, for some reason.

You can use flex-box for the purpose.
https://www.w3schools.com/css/css3_flexbox.asp

Reply:
Also, your ul element has padding.
Use the style to align to center:
```css
ul {
  list-style: none;
  padding: 0;
}
```

Reply:
So that's why "list-style: none;" didn't seem to work! I had to set its padding to 0.

Reply:
:slightly_smiling_face:

Reply:
You can detect the things by browser "Inspect".
I can teach you if you'd like.